composer.json

```diff
@@ -22,12 +22,12 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "barryvdh/reflection-docblock": "^2.0.6",
+        "barryvdh/reflection-docblock": "^2.1.1",
         "composer/class-map-generator": "^1.0",
-        "doctrine/dbal": "^2.6 || ^3.1.4",
-        "illuminate/console": "^9 || ^10",
-        "illuminate/filesystem": "^9 || ^10",
-        "illuminate/support": "^9 || ^10",
+        "illuminate/console": "^10",
+        "illuminate/database": "^10.38",
+        "illuminate/filesystem": "^10",
+        "illuminate/support": "^10",
```

Comment:
Since we removed the dependency on Laravel 9, we can remove `^7` from `orchestra/testbench` located in `require-dev`.